newsfragments/46944.significant.rst

```diff
@@ -0,0 +1,3 @@
+Statically-coded executors are now deprecated
```

Comment:
We need here also announcement that hybrid executor feature is now stable.

Reply:
Good call. I've updated the newsfragment to make that more clear!